packages/@ourworldindata/grapher/src/timeline/TimelineComponent.tsx

```diff
@@ -358,13 +376,21 @@ const TimelineHandle = ({
                 left: `${offsetPercent}%`,
             }}
         >
-            <Tippy
-                content={<span>{tooltipContent}</span>}
-                visible={tooltipVisible}
-                zIndex={tooltipZIndex}
-            >
-                <div className="icon" />
-            </Tippy>
+            <div className="icon" />
+            {tooltipVisible && (
+                <>
+                    <div
+                        className="handle-label-arrow"
+                        style={{ zIndex: tooltipZIndex }}
+                    />
+                    <div
+                        className="handle-label"
+                        style={{ zIndex: tooltipZIndex }}
+                    >
+                        {tooltipContent}
+                    </div>
+                </>
+            )}
```

Comment:
Note for the reviewers: I'm rolling my own tooltips instead of relaying on Tippy since I wanted these to look exactly like the tooltips used for the action buttons.